src/renderer/survey/js/webview.js

```diff
@@ -0,0 +1,132 @@
+/**
+ * The preload script for the survey webview
```

Comment:
Can you think of any way that the contents of this file could be tested? Perhaps via some kind of bare-bones "mock survey server"?

Reply:
There is no need for a mock HTTP server in order to test the webview preload script. The basic idea of the approach which we have used can be found in the following JIRA ticket: https://issues.gpii.net/browse/GPII-2870.